core/src/main/java/com/cloud/storage/resource/StorageSubsystemCommandHandlerBase.java

```diff
@@ -142,7 +142,7 @@ protected Answer execute(CreateObjectCommand cmd) {
             }
             return new CreateObjectAnswer("not supported type");
         } catch (Exception e) {
-            logger.debug("Failed to create object: " + data.getObjectType() + ": " + e.toString());
+            logger.error(String.format("Failed to create object [%s] due to [%s].", data.getObjectType(), e), e);
```

Comment:
as this is targeted for main, shouldn't we use log4j2 string formatting? i.e.
```suggestion
            logger.error("Failed to create object [{}] due to [{}].", data.getObjectType(), e.getLocalizedMessage(), e);
```